lexicons/app/bsky/graph/subscribeMuteList.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.subscribeMuteList",
```

Comment:
Not sure if it's a material issue, but flagging for good measure— `subscribe` also may end-up being a prefix for xrpc subscription methods which could be a naming conflict, i.e. `subscribeRepos`.

Reply:
One alternative could be to call these `addMuteList` and `removeMuteList`.

Reply:
that's a good point 👍  i'll change it